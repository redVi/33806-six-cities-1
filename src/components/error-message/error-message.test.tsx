import React from 'react';
import renderer from 'react-test-renderer';
import ErrorMessage from '@/components/error-message/error-message';

describe(`ErrorMessage`, () => {
  it(`renders correctly`, () => {
    const tree = renderer.create(<ErrorMessage error="You failed" />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
