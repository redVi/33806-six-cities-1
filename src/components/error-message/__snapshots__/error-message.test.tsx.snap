// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ErrorMessage renders correctly 1`] = `
<span
  className="error error__validation"
>
  You failed
</span>
`;
